drivers/overlay/overlay.go

```
@@ -370,6 +384,38 @@ func parseOptions(options []string) (*overlayOptions, error) {
 				}
 				o.imageStores = append(o.imageStores, store)
 			}
+		case "additionallayerstore":
+			logrus.Debugf("overlay: additionallayerstore=%s", val)
+			// Additional read only layer stores to use for lower paths
+			if val == "" {
+				continue
+			}
+			for _, store := range strings.Split(val, ",") {
+				elems := strings.Split(val, ":")
+				store = filepath.Clean(elems[0])
+				if !filepath.IsAbs(store) {
+					return nil, fmt.Errorf("overlay: additionallayerstore path %q is not absolute.  Can not be relative", store)
+				}
+				st, err := os.Stat(store)
+				if err != nil {
+					return nil, fmt.Errorf("overlay: can't stat additionallayerstore dir %s: %v", store, err)
+				}
+				if !st.IsDir() {
+					return nil, fmt.Errorf("overlay: additionallayerstore path %q must be a directory", store)
+				}
+				var withReference bool
+				for _, e := range elems[1:] {
+					if e == "ref" {
+						withReference = true
```

TO be anal here, we could return an error `if withReference == true`